tests/gauge_alg_test.cpp

```diff
@@ -451,32 +393,11 @@ int main(int argc, char **argv)
   if (gauge_load) {
     printfQuda("Loading gauge field from host\n");
     for (int dir = 0; dir < 4; dir++) {
-      host_gauge[dir] = safe_malloc(V * gauge_site_size * host_gauge_data_type_size);
+      host_gauge[dir].resize(V * gauge_site_size * host_gauge_data_type_size);
```

Comment:
No particular reason why this isn't pinned, it was always in pageable memory and I didn't feel like changing it.

Agree to some degree with respect to some of you comments here.  Eventually we'll have some cleaner and more natural for allocating memory in the tests.  I made the change here because, IIRC, it fixed an issue.